mobify/platform-scaffold
# Offline state in SDK

This PR replaces the offline store keys and their corresponding actions and selectors with those in the linked SDK PR. 

 **JIRA**: Preliminary for https://mobify.atlassian.net/browse/WEBDATA-22
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/677

## Changes
- Remove the existing actions, selectors, and reducer
- Import and use the ones from the SDK

## How to test-drive this PR
- Link with the corresponding SDK PR
- Preview Merlin's Potions
- Navigate to a PLP
- Set to offline in the dev tools
- Navigate back to home, seeing that the page loads and has the offline banner
- Navigate to a different PLP
- See that the offline screen is shown.
- See that the Redux store has relevant information in the `offline` branch.
